README.md

```diff
@@ -37,4 +37,14 @@ Let hbac do its magick 😊
 
 or
 
-use `require("hbac").close_unused()` to close all unedited buffers
+- `:Hbac close_unused` - close all unedited buffers
+- `:Hbac pin` - pin the current buffer and prevent it from being auto-closed
+- `:Hbac unpin` - unpin the current buffer and allow it to be auto-closed
+
+or, if you prefer to use lua:
+
+```lua
+require("hbac").hbac_cmd("pin")
+require("hbac").hbac_cmd("unpin")
+require("hbac").hbac_cmd("close_unused")
+```
```

Comment:
Would you prefer `.close_unused()`, `.pin()`, and `.unpin()`, or some other names, for that lua structure?